File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/DbsBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author yudong
 * @date 2023/2/15
 */
@NoArgsConstructor
@Data
public class DbsBean {

    private String name;
    private String select;
    private String key;
    private Boolean defaultDB;
    private PropertiesBean properties;

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/DbsBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/15
 */


@AllOpen
class DbsBean {
    var name: String? = null
    var select: String? = null
    var key: String? = null
    var defaultDB: Boolean? = null
    var properties: PropertiesBean? = null
}
